scripts/install/ceres_install.bash

```diff
@@ -2,6 +2,7 @@
 set -e  # exit on first error
 UBUNTU_VERSION=`lsb_release --release | cut -f2`
 SRC_PREFIX_PATH="/usr/local/src/"
```

Comment:
Yeah I have come around that /usr/local/src is a correct place to put source code. Still not sure if this script should do it. In autonomoose we just put the builds into `tmp`. The reasoning being someone running the "install_dependencies" script just wants dependencies installed, and didn't ask for or expect source code in particular. Indeed most of the scripts just run `apt-get`, and we could change this ceres script to use `apt-get` tomorrow.

.. I agree with your comment elsewhere that "we should maybe create Ubuntu packages and use apt-get install"